.github/CODEOWNERS

```
@@ -278,6 +287,8 @@
 /packages/problemchild @elastic/ml-ui @elastic/sec-applied-ml
 /packages/prometheus @elastic/obs-infraobs-integrations
 /packages/prometheus/data_stream/remote_write @elastic/obs-cloudnative-monitoring
+/packages/prometheus/data_stream/collector @elastic/obs-cloudnative-monitoring
```

```suggestion
/packages/prometheus/data_stream/collector @elastic/obs-infraobs-integrations
```

This belongs to Infraobs team

---

ty for catching that 🙂